### ISSUE TITLE
[BUG] Unable to download entire course

### ISSUE THREAD

**Issue by aerobo** [nishad/udemy-dl#187]
Please prefix your issue with one of the following: [BUG], [PROPOSAL], [QUESTION].

Replace everything between stars with current version of your udemy-dl (output of udemy-dl -v):  
Version : **0.2.2-alpha**

**In raising this issue, I confirm the following (please check boxes, eg [X]):**

- [x] I have read and understood the [contributors guide](https://github.com/nishad/udemy-dl/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [x] I have checked that the issue I'm posting isn't already reported.
- [x] I have checked that the issue I'm porting isn't already solved and no duplicates exist in [closed issues](https://github.com/nishad/udemy-dl/issues?q=is%3Aissue+is%3Aclosed) and in [opened issues](https://github.com/nishad/udemy-dl/issues)
- [x] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

**My familiarity with the project is as follows (check one, eg [X]):**

- [ ] I have never used the project.
- [ ] I have used the project briefly.
- [x] I have used the project extensively, but have not contributed previously.
- [ ] I am an active contributor to the project.

**If the project maintainer has any additional requirements, they will be listed here.**

- No additional requirements.

---

Having issues downloading the full course from udemy which is  https://www.udemy.com/complete-networking-fundamentals-course-ccna-start/
It will stop downloading at number 470 out of 538. I have used advanced commands and such to no avail.

**Comment by nishad** [comment]
**All issues should contain mandatory check lists, version number and proper prefix for the subject line. Issues reported without checklists, version or without proper prefix will be closed or ignored.**

Due to this policy I'm closing this thread, you can request to reopen this issue after making sufficient changes.

**Comment by aerobo** [comment]
Ok, I have edited it to what I believe is the correct way, sorry it's my first time posting. Would you be willing to reopen this issue again?

**Comment by nishad** [comment]
Could you provide last error messages showing in the console window ?

**Comment by aerobo** [comment]
@nishad  It doesn't show an error message but just shows that it's completed downloading everything, as shown here. There are more than 470 parts to the course which it's not continuing to download but logs out instead.      [https://ibb.co/i1D28v](url)

**Comment by nishad** [comment]
Could you try with    ?

```
--lecture-start LECTURE_START
                            Lecture to start at (default is 1)
--lecture-end LECTURE_END
                            Lecture to end at (default is last)
```

**Comment by aerobo** [comment]
Yes, I tried this, and the end result was this....  it seemed to only download a few files that were tests, but did not recognize to continue the course.
![bug3](https://cloud.githubusercontent.com/assets/25488883/22852264/5d947d48-f006-11e6-96ae-6aa3c8d5c786.png)

**Comment by nishad** [comment]
What was the command you were trying ? (remove credentials and course link)

**Comment by aerobo** [comment]
@nishad  udemy-dl -u user -p password --lecture-start 472 --lecture-end 538 https://www.udemy.com/course name

**Comment by aerobo** [comment]
@nishad is there anything else I should try?

**Comment by nishad** [comment]
I am still investigating this issue and couldn't reproduce the same with a comparatively larger course I had enrolled. I will wait for more user inputs. Meanwhile you can upload a 'debug' file to help me identify, what exactly went wrong with this specific course.

**Comment by aerobo** [comment]
@nishad  in which way should I go about obtaining and producing a debug file?

**Comment by aerobo** [comment]
@nishad I was unable to produce any debug file when running the command unfortunately.

**Comment by aerobo1** [comment]
@nishad Hey this is aerobo, just made a new account because I forgot my credentials for the last one, here's the debug files for you. 

http://www.mediafire.com/file/3mo9b32jew0imqu/debug+udemy+dl.tar.gz

**Comment by Spartacode** [comment]
I have a similar issue. Stops download at video 495.
